data/sles4sap/qe_sap_deployment/qesap_azure.yaml

```diff
@@ -0,0 +1,32 @@
+provider: "%PROVIDER%"
+apiver: 1
+terraform:
+  variables:
+    az_region: "%REGION%"
+    deployment_name: "%DEPLOYMENTNAME%"
+    os_image: "%TRENTO_CLUSTER_OS_VER%"
+    private_key: "%SSH_KEY_PRIV%"
+    public_key: "%SSH_KEY_PUB%"
+
+ansible:
+  hana_urls:
+    - "%HANA_SAR%"
+    - "%HANA_CLIENT_SAR%"
+    - "%HANA_SAPCAR%"
+  create:
+    - registration.yaml -e reg_code=${REG_CODE} -e email_address=${EMAIL}
+    - pre-cluster.yaml
+    - sap-hana-preconfigure.yaml -e use_sapconf=${SAPCONF}
```

Comment:
I'd drop the use_sapconf option.  The default is saptune which is the better solution.  By default, saptune code will apply HANA tuning.